x/staking/autocli.go

```diff
@@ -0,0 +1,140 @@
+package staking
+
+import (
+	"fmt"
+
+	autocliv1 "cosmossdk.io/api/cosmos/autocli/v1"
+	v1beta "cosmossdk.io/api/cosmos/staking/v1beta1"
+
+	"github.com/cosmos/cosmos-sdk/version"
+)
+
+func (am AppModule) AutoCLIOptions() *autocliv1.ModuleOptions {
+	return &autocliv1.ModuleOptions{
+		Query: &autocliv1.ServiceCommandDescriptor{
+			Service: v1beta.Query_ServiceDesc.ServiceName,
+			RpcCommandOptions: []*autocliv1.RpcCommandOptions{
+				{
+					RpcMethod: "Validators",
```

Comment:
This gives me `proto: google.protobuf.Any: unable to resolve \"/cosmos.crypto.ed25519.PubKey\": not found`.